### PR TITLE
pythonPackages.vcard: init at 0.13.0

### PR DESCRIPTION
**Pull request by l0b0**
###### Motivation for this change

Add vCard validator.

###### Things done

- [x] Tested using sandboxing on NixOS
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests)) (Not applicable AFAICT)
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"` (Not applicable)
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after *(before and after building? How to do this if I was not aware of this package before building?)*)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @l0b0

### CODE REVIEW

**Review by FRidh — CHANGES_REQUESTED** [review]
If it is used as a library the expresssion needs to be moved under `python-modules/`. In that case also explicitly list all dependencies as arguments, instead of passing in `python3Packages`. Currently you are mixing both the package set and dependencies and that should especially be avoided.

Aside from that it looks good.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3039,6 +3039,16 @@
     github = "lnl7";
     name = "Daiderd Jordan";
   };
+  l0b0 = {
```
Comment (alexarice) [inline maintainers/maintainer-list.nix]: Can the addition to `maintainer-list.nix` be in a separate commit?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -6157,6 +6157,8 @@ in {
 
   stringcase = callPackage ../development/python-modules/stringcase { };
 
+  vcard = callPackage ../tools/text/vcard { };
```
Comment (jonringer) [inline pkgs/top-level/python-packages.nix]: I would move the expression under python-modules, and then just add
```
disabled = isPy27;
```
to show that it's only python3 supported.

This would also mean removing python3Packages and importing the dependencies you need directly by name.

Comment (l0b0) [reply pkgs/top-level/python-packages.nix]: Thank you for reviewing! After adding `disabled = isPy27` and changing `with python3Packages` to `with pythonPackages` (because otherwise it's specified twice that it's a Python 3 package) I get this error:

> error: vcard-0.13.0 not supported for interpreter python2.7

Is this intentional? Do I need to specify twice that it's Python 3 only?

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (l0b0) [reply maintainers/maintainer-list.nix]: Is this OK?

**Comment by l0b0** [comment]
> If it is used as a library the expresssion needs to be moved under `python-modules/`. In that case also explicitly list all dependencies as arguments, instead of passing in `python3Packages`. Currently you are mixing both the package set and dependencies and that should especially be avoided.

Thank you for reviewing! Can you please review my changes?

This package is meant to be used both as a command line tool (in fact that's the only way I use it myself), but it's also been made to work as a library. I've tried to make that work, but I'm not sure this is the right way of going about it. The documentation seems to focus on doing either one or the other, not both.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (alexarice) [reply maintainers/maintainer-list.nix]: It would be better if the commit was named `Add l0b0 to maintainers.nix` but that is the right structure

Comment (l0b0) [reply maintainers/maintainer-list.nix]: Meet you halfway? :)

Comment (alexarice) [reply maintainers/maintainer-list.nix]: That's great, thanks

File: pkgs/top-level/python-packages.nix (hunk 2 above)
Comment (alexarice) [reply pkgs/top-level/python-packages.nix]: I believe that error is because when you use `pythonPackages`, `toPythonApplication` tries to check that it can run your application with python 2, sees that this is disabled, then runs an error.

**Review by alexarice — APPROVED** [review]
Builds via nix-review and the resulting program runs though I can't really test it as I have no idea what a vcard is

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (jonringer) [reply maintainers/maintainer-list.nix]: generally I see "maintainers: add l0b0"

https://github.com/NixOS/nixpkgs/commits/6f1445ce9eee3ee3e7efe5bedd5c2a4718671c30/maintainers/maintainer-list.nix

Comment (alexarice) [reply maintainers/maintainer-list.nix]: that looks familiar and is what I was trying to remember for my first suggestion I think

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM (good job!)
binary seems to work
leaf package

Personally I would rename the maintainer commit to `maintainer: add l0b0` and then you look good to me. :)

**Comment by l0b0** [comment]
> […] I can't really test it as I have no idea what a vcard is

[RFC 2426](https://tools.ietf.org/html/rfc2426). Unfortunately the standard is in part ambiguous and self-contradictory, and it's really hard to find valid examples in the wild – even the examples given in section 7 *of the RFC* are invalid - they are missing the mandatory `N` property.

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (l0b0) [reply maintainers/maintainer-list.nix]: And done.

**Comment by l0b0** [comment]
Welp. Moved the changes to a branch, and now I don't seem to be able to recover this PR.
